README.md

```diff
@@ -786,6 +779,20 @@ A regexp pattern string that is matched against all test paths before executing
 
 This is useful if you need to override the default. If you are testing one file at a time the default will be set to `/.*/`, however if you pass a blob rather than a single file the default will then be the absolute path of each test file. The override may be needed on windows machines where, for example, the test full path would be `C:/myproject/__tests__/mystest.jsx.jest` and the default pattern would be set as `/C:\myproject\__tests__\mystest.jsx.jest/`.
 
+### `testRegex` [string]
+(default: `'__tests__/.*\.js
+
+## Local Development
```

Comment:
ugh, it must be a bug in the README generation script that probably matches on the default testRegex… I'll fix it sometime next week, thanks for noticing.